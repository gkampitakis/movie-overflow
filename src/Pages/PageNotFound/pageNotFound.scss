.page_not_found {
  text-align: center;

  img {
    user-select: none;
    user-select: none;
    -moz-user-select: none;
    -webkit-user-drag: none;
    -webkit-user-select: none;
    -ms-user-select: none;
  }
}
