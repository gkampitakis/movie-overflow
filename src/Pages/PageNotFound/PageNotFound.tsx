import React from 'react';
import NotFoundLogo from '../../assets/404.svg';
import './pageNotFound.scss';

export default function PageNotFound() {
  return (
    <section className="main_content page_not_found">
      <img src={NotFoundLogo} alt="Page not found" />
    </section>
  );
}
