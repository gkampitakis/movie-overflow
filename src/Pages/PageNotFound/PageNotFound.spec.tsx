import React from 'react';
import PageNotFound from '.';
import { render } from '@testing-library/react';

describe('PageNotFound', () => {
  it('Should render consistently', () => {
    const { container } = render(<PageNotFound />);

    expect(container).toMatchSnapshot();
  });
});
