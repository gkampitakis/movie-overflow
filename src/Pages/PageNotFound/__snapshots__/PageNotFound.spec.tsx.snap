// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Should render consistently 1`] = `
<div>
  <section
    class="main_content page_not_found"
  >
    <img
      alt="Page not found"
      src="404.svg"
    />
  </section>
</div>
`;
